website/docs/essays/tx-pool-rbf.md

```diff
@@ -11,21 +11,29 @@ import Link from "@docusaurus/Link";
 
 When a transaction is broadcasted to the network, all nodes verify the transaction and add it to their transaction pool (tx-pool) if it is valid. The transaction will stay in the tx-pool until it is mined into a block.
 
-There is a scenario that a transaction maybe stuck in the tx-pool for a long time because the transaction fee is not high enough. In this case, the transaction can be replaced by a new transaction with a higher fee. This strategy is called Replace-By-Fee (RBF), RBF was first introduced by Bitcoin in [BIP: 125](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki).
+There is a scenario that a transaction maybe stuck in the tx-pool for a long time because the transaction fee is not high enough. In this case, the transaction can be replaced by a new transaction contains same inputs but with a higher fee. This strategy is called Replace-By-Fee (RBF), RBF was first introduced by Bitcoin in [BIP 125](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki).
 
-In most cases, RBF is usefull for spenders to adjust their previously-sent transactions to deal with unexpected confirmation delays or to perform other useful replacements, such as merging multiple transactions into one.
-
-The new transaction must have the at least one same inputs as the old one(may includes other extra inputs), and the outputs can be different. The new transaction will replace the old one in the tx-pool, the old one will be removed from the tx-pool and with the status Rejected.
+The new transaction must have the at least one same inputs as the old one(may includes other extra inputs), and the outputs can be different. The new transaction will replace the old one in the tx-pool, the old one will be removed from tx-pool with status Rejected.
```

Comment:
The new transaction must have at least one input that is the same as the old one (may include extra inputs), while the outputs can be different. The new transaction will replace the old one in the tx-pool, and the old one will be removed from the tx-pool with the status "Rejected."